CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ browser compatible release will be made soon.
 ## 4.0.0 - 2019-01-30
 
 ### Changed
+- **BREAKING**: Uses `v1.driver(options)` API instead of `v1.veres()`.
```

Comment:
Is this supposed to be under an upcoming release heading?

Reply:
Oh, hmm, that should probably be changed to v6, yeah?

Reply:
6.0 is already released, this would be 7.0.  Which makes me wonder if this whole branch needs to be rebased?

Reply:
I just rebased this branch on the latest master yesterday, so it should be fine.
(And yeah, I'll change it to v7. I was confused looking at the 6.0.1 thing)

Reply:
Fixed.